scripts/templates/deploy.template.js

```diff
@@ -0,0 +1,21 @@
+const Witnet = artifacts.require("Witnet")
+const WitnetRequestBoardProxy = artifacts.require("WitnetRequestBoardProxy")
+const #artifact = artifacts.require("#artifact")
+
+const addresses = {
```

Comment:
You could import directly the addresses from the file. :)

Reply:
Yep, so right. Thnx!